src/pmem.c

```diff
@@ -435,6 +447,45 @@ pmem_msync(void *addr, size_t len)
 }
 
 /*
+ * pmem_align -- align destination address on 16 byte boundary.
+ */
+static void
```

Comment:
IMHO, it would be better to return aligned address by value, not via reference.
```C
static void *
pmem_align(void *dest, const void *src, size_t *cnt, int direction)
...
void *aligned_dest = pmem_align(dest, src, cnt, UP);
```